samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Drivers/WebDriverDriver.cs

```diff
@@ -14,7 +14,7 @@ public class WebDriverDriver : IDisposable
         private static IWebDriver SetupWebDriver()
         {
             var options = new ChromeOptions();
-            options.AddArgument("--headless");
+            //options.AddArgument("--headless");
```

Comment:
Should this remain in the code commented?

Reply:
yes, good catch 👍